R/IncidenceRates.R

```diff
@@ -56,15 +56,13 @@ getIncidenceRate <- function(connectionDetails = NULL,
   }
   
   ParallelLogger::logInfo("Calculating incidence rate per year by age and gender")
-  sql <-
-    SqlRender::loadRenderTranslateSql(
-      sqlFilename = "GetCalendarYearRange.sql",
-      packageName = utils::packageName(),
-      dbms = connection@dbms,
-      cdm_database_schema = cdmDatabaseSchema
-    )
+  sql <- SqlRender::readSql(system.file("sql/sql_server/GetCalendarYearRange.sql", 
```

Comment:
Code changes in this file are unrelated to the PR. Doing this increases the chances of a bug entering in the codebase

Reply:
It can - but we can test it. Reason i am making the change here is to ensure we have the same pattern everywhere. Consistent pattern will help us debug. This pattern is being used everywhere